src/experimental/keycloak/charts/postgresql/.helmignore

```diff
@@ -0,0 +1,2 @@
+.git
```

Comment:
no need to submit dependencies (charts/postgresql), just declare them in requirements.yaml and the bot will download and pack into automatically.